src/language-js/printer-estree.js

```diff
@@ -3418,30 +3418,31 @@ function shouldGroupFirstArg(args) {
   );
 }
 
-const functionCompositionFunctionNames = {
-  pipe: true, // RxJS, Ramda
-  pipeP: true, // Ramda
-  pipeK: true, // Ramda
-  compose: true, // Ramda, Redux
-  composeFlipped: true, // Not from any library, but common in Haskell, so supported
-  composeP: true, // Ramda
-  composeK: true, // Ramda
-  flow: true, // Lodash
-  flowRight: true, // Lodash
-  connect: true // Redux
-};
+const functionCompositionFunctionNames = new Set([
```

Comment:
Hehe, I proposed using a Set in the original PR :)

https://github.com/prettier/prettier/pull/4431#discussion_r186300367